index.bs

```diff
@@ -839,6 +838,20 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  Let |attestationPresentationPreference| be the value of |options|.{{MakePublicKeyCredentialOptions/attestation}}.
+                    <dl class="switch">
+                    <dt>If the value of |attestationPresentationPreference| is "none" (or it has been left unspecified),</dt>
+                    <dd>replace potentially identifying information (such as AAGUID and attestation certificates) 
+                        in the [=attested credential data=] and [=attestation statement=] with blinded versions of the same data.</dd>
+
```

Comment:
seems like we ought to have impl cons info somewhere in the spec that provides guidance wrt performaing such blinding.  I suppose that could be done at some point post-WD-07